plugins/cc/ccalgo.ml

```diff
@@ -27,7 +27,8 @@ let init_size=5
 let cc_verbose=ref false
 
 let print_constr t =
-  let sigma, env = Pfedit.get_current_context () in
+  let env = Global.env () in
```

Comment:
wrong env

Reply:
@ejgallego why not simply call `Vernacstate.Proof_global.get_current_context` here?